File: pearls/col1/java/prob1.java
import java.util.*;

public class prob1 {
    public static void main(String[] args) {
	SortedSet<Integer> nums = new TreeSet<Integer>();
	Scanner scan = new Scanner(System.in);
	while (scan.hasNextInt()) {
	    nums.add(scan.nextInt());
	}
	for (Integer num : nums) {
	    System.out.println(num);
	}
    }
}

File: pearls/col1/prob1.go
package main

import (
    "fmt"
    "sort"
    "os"
)

const COUNT = 10000000

func main() {
    nums := make([]int, COUNT)
    count := 0
    for count < COUNT {
        var num int
        n, err := fmt.Scan(&num)
        if err == os.EOF { break }
        if n > 0 {
            nums[count] = num
            count++
        }
    }

    sort.Ints(nums[:count])
    for i := 0; i < count; i++ {
        fmt.Println(nums[i])
    }
}
